docs/getting-started.md

```diff
@@ -124,6 +169,72 @@ By providing a separate snapshot ID than the container ID we can easily reuse, e
 
 We also add a line to delete the container along with its snapshot after we are done with this example.
 
+Here is example code to pull redis image based on alpine linux from Dockerhub, create an OCI spec, create a container based on the spec and finally delete the container.
```

Comment:
nit:  "pull redis image" to "pull the redis image".   (BTB, this is a good synopsis for this example).